### DISCUSSION TITLE
[Feature] Database automatically backup custom location setting

### DISCUSSION THREAD

**Discussion by mcjak97** [immich-app/immich#14136] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello.
It will be good to have a setting on the backup feature with the custom location of database backup, for example, to save to a mounted NAS location, not on the local host.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can mount the backup folder to wherever you want. See https://immich.app/docs/guides/custom-locations

**Reply by mcjak97** [reply]
Strange is that when i apply these settings (bold settings), server will not start, why?
I have write these settings:
**docker-compose.yml**
immich-server:
      - ${LIBRARY_LOCATION}:/usr/src/app/upload/library
      - ${UPLOAD_LOCATION}:/usr/src/app/upload/upload
      - ${THUMBS_LOCATION}:/usr/src/app/upload/thumbs
      - ${PROFILE_LOCATION}:/usr/src/app/upload/profile
      - ${VIDEO_LOCATION}:/usr/src/app/upload/encoded-video
      **- ${BACKUP_LOCATION}:/usr/src/app/upload/backups**

**.env**
LIBRARY_LOCATION=/mnt/photos
THUMBS_LOCATION=/mnt/immich-cache/thumbs
UPLOAD_LOCATION=/mnt/immich-cache/upload
PROFILE_LOCATION=/mnt/immich-cache/profile
**BACKUP_LOCATION=/mnt/immich-cache/backups**

**Reply by bo0tzz** [reply]
Presumably you didn't move the files to these new locations.

**Reply by mcjak97** [reply]
I just have added backup_location to config files, those settings was already in place before this.
It is not needed to move files.

**Reply by bo0tzz** [reply]
> It is not needed to move files.

Yes it is. The error that the server is logging when it fails to start will tell you the same ;)

**Reply by mcjak97** [reply]
Wait a minute, what files are we talking about?
I don't change anything about files location, just added backup_location lines to config files. I am confused now.

**Reply by bo0tzz** [reply]
Ah, I see now that you don't have a mount for the "root" `/usr/src/app/upload`. We don't recommend doing that. See https://immich.app/docs/guides/custom-locations for the setup we suggest.
